README.md

```diff
@@ -55,7 +55,7 @@ currently emitted:
 | Y019 | Certain kinds of methods should use `_typeshed.Self` instead of defining custom `TypeVar`s for their return annotation. This check currently applies for instance methods that return `self`, class methods that return an instance of `cls`, and `__new__` methods.
 | Y020 | Quoted annotations should never be used in stubs.
 | Y021 | Docstrings should not be included in stubs.
-| Y022 | Imports linting: use typing-module aliases to stdlib objects as little as possible (e.g. `builtins.list` over `typing.List`, `collections.Counter` over `typing.Counter`, etc.).
+| Y022 | Use typing-module aliases to stdlib objects as little as possible (e.g. `builtins.list` over `typing.List`, `collections.OrderedDict` over `typing_extensions.OrderedDict`, etc.).
```

Comment:
Why use `typing_extensions` here? I think `typing` is more "canonical", easier to understand, and more succinct.

Reply:
I realised we didn't mention anywhere that Y022 bans various imports from `typing_extensions` as well as various imports from `typing`, and that felt like something which might be useful to make clear. But I can reword this if the current phrasing is just confusing!

Reply:
How's it look after 1bf878b?